chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -131,33 +124,28 @@ pub fn bridge_hub_kusama_session_keys(keys: AuraId) -> bridge_hub_kusama_runtime
 
 // AssetHubPolkadot
 fn asset_hub_polkadot_genesis(
-	wasm_binary: &[u8],
 	invulnerables: Vec<(AccountId, AssetHubPolkadotAuraId)>,
 	endowed_accounts: Vec<AccountId>,
 	id: ParaId,
-) -> asset_hub_polkadot_runtime::RuntimeGenesisConfig {
-	asset_hub_polkadot_runtime::RuntimeGenesisConfig {
-		system: asset_hub_polkadot_runtime::SystemConfig {
-			code: wasm_binary.to_vec(),
-			..Default::default()
-		},
-		balances: asset_hub_polkadot_runtime::BalancesConfig {
+) -> serde_json::Value {
+	serde_json::json!({
+		"balances": asset_hub_polkadot_runtime::BalancesConfig {
 			balances: endowed_accounts
 				.iter()
 				.cloned()
-				.map(|k| (k, ASSET_HUB_POLKADOT_ED * 4096))
+				.map(|k| (k, ASSET_HUB_POLKADOT_ED * 4096 * 4096))
```

Comment:
intentional increase?

Reply:
These are just genesis configs for testing, so changing them is fine.

Reply:
yes, but still might affect tests, wanted to point in case of accidental change